test/resources/class_with_factory_constructors.dart

```diff
@@ -0,0 +1,22 @@
+class Logger {
+  final String name;
+  bool mute = false;
+
+  // _cache is library-private, thanks to
+  // the _ in front of its name.
```

Comment:
Do we need this comment in tests?

Reply:
Yes. This is good example class for future testing some other cases.